app/services/user_alerts/alert_user_about_new_device.rb

```diff
@@ -61,7 +61,7 @@ def self.sign_in_events_start_time(user:)
       # buffer is added to account for delays of the job run or within the job itself.
       [
         user.sign_in_new_device_at,
-        (IdentityConfig.store.new_device_alert_delay_in_minutes * 3).minutes.ago,
+        (IdentityConfig.store.new_device_alert_delay_in_minutes * 2).minutes.ago,
```

Comment:
Why are we updating this?

Reply:
I'm not 100% certain how critical this but in some cases the original sign in event is coming through after fail - success.
![Screenshot 2024-05-31 at 11 11 41 AM (2)](https://github.com/18F/identity-idp/assets/135744319/8e831ed2-0eb7-40a6-af1a-6bb45e22b1f3)

If that's OK and not really a problem I would leave it at t * 3 minutes ago.

Reply:
Hm I'm not sure I follow the problem you're seeing. Do you have some steps to reproduce?

Reply:
Steps in the testing plan. 
1. sign in
2. wait to MFA for timeout event to send an email
3. fail MFA
4. succeed MFA

Maybe this is a non-issue since at least it's definitely limited by https://github.com/18F/identity-idp/blob/29630cfd4c232cf51c242fd3ca204d584725eaea/app/services/user_alerts/alert_user_about_new_device.rb#L65-L66
I guess I'm unclear about how stale is too stale.
![Screenshot 2024-06-03 at 9 58 09 AM (2)](https://github.com/18F/identity-idp/assets/135744319/02a79469-0674-4148-b88f-ba5ff0b659de)